install_config/install/docker_registry.adoc

```diff
@@ -169,9 +170,68 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+OpenShift refers to the integrated registry by its service IP address,
+so if you decide to delete and recreate the docker-registry service,
```

Comment:
I agree w/ the asterisk suggestion, at least as far as matching what we (mostly) use elsewhere for "unique names of objects".

Side note: I wish we used backticks more liberally in our markup guidelines, like for values of parameters/settings, and possibly for general "names of objects" like these. :disappointed: